akvo/rsr/static/tests/scripts-src/my-results/utils.test.js

```diff
@@ -85,59 +102,69 @@ describe("arrowFunction isNewUpdate", () => {
 describe("function collapseId", () => {
     it('expect to create an collapsedId from the model name and parent object id', () => {
         const model = 'model2',
-              id = 200;
+            id = 200;
         expect(utils.collapseId(model, id)).to.equal('model2-200');
     });
 });
 
-// requires that the function tested is changed to an exported function
 describe("function childModelName", () => {
     it('expect to return child model name', () => {
         const model = c.OBJECTS_RESULTS,
-              child = c.OBJECTS_INDICATORS;
+            child = c.OBJECTS_INDICATORS;
         expect(utils.childModelName(model)).to.equal(child);
     });
 });
 
 describe("function parentModelName", () => {
     it('expect to return parent model name', () => {
         const model = c.OBJECTS_INDICATORS,
-              parent = c.OBJECTS_RESULTS;
+            parent = c.OBJECTS_RESULTS;
         expect(utils.parentModelName(model)).to.equal(parent);
     });
 });
 
 describe("function levelAbove", () => {
     it('expect to compare index of array to decide that level is above', () => {
-        const models = [1, 2],
-              model = models[1],
-              compare = models[0];
+        const model = c.OBJECTS_PERIODS,
+            compare = c.OBJECTS_UPDATES;
         expect(utils.levelAbove(model, compare)).to.equal(model < compare);
     });
 });
 
 describe("function levelBelow", () => {
     it('expect to compare index of array to decide that level is below', () => {
-        const models = [1, 2],
-              model = models[0],
-              compare = models[1];
+        const model = c.OBJECTS_UPDATES,
+            compare = c.OBJECTS_PERIODS;
         expect(utils.levelBelow(model, compare)).to.equal(model > compare);
     });
 });
 
-// requires that the function tested is changed to an exported function
+describe("function fullUpdateVisibility", () => {
```

Comment:
While the test works, it doesn't really test the function. That's not your fault however for not understanding the fairly complex logic of fullUpdateVisibility().

Pleas add a comment saying **TODO: update test so we test the full functionality of  fullUpdateVisibility()** so we can look at it later.